src/Controller/Controller.php

```diff
@@ -728,9 +728,9 @@ public function viewClasses(): array
      * Each view class must implement the `getContentType()` hook method
      * to participate in negotiation.
      *
-     * @param array $viewClasses View classes list.
+     * @param array<string> $viewClasses View classes list.
```

Comment:
Think we can use LIst<string> now.

Reply:
True, we can update accordingly.

Reply:
I found also a few others
There are potentially 87 other occurances left, but thats maybe for a follow up PR.